src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -677,6 +678,9 @@ public static T RunImpersonated<T>(SafeAccessTokenHandle safeAccessTokenHandle,
             return result;
         }
 
+        public static Task RunImpersonatedAsync(SafeAccessTokenHandle safeAccessTokenHandle, Func<Task> func) => RunImpersonated(safeAccessTokenHandle, func);
+
+        public static Task<T> RunImpersonatedAsync<T>(SafeAccessTokenHandle safeAccessTokenHandle, Func<Task<T>> func) => RunImpersonated(safeAccessTokenHandle, func);
```

Comment:
Can you chop these (either before or after the arrow)? It's nice when code fits in the default github code window.

Reply:
done